test/elpa.el

```diff
@@ -0,0 +1,5 @@
+(setq package-user-dir
+      (expand-file-name (format ".cask/%s/elpa" emacs-version)))
```

Comment:
> Does Cask still have problem being used from ansi-term? That was another issue I had earlier.

Guess @lunaryorn or @rejeep can answer this.

I'm fine with dropping cask in general, but I liked the idea of having some solution that makes it easy to have an uniform test setup across different projects.